viewer/packages/cad-geometry-loaders/src/CadModelUpdateHandler.ts

```diff
@@ -117,7 +117,6 @@ export class CadModelUpdateHandler {
 
   updateCamera(camera: THREE.PerspectiveCamera): void {
     this._cameraSubject.next(camera);
-    this._progressSubject.next(notLoadingState);
```

Comment:
Do we know why this was added in the first place as it does not make any sense to block the loading indicator while the camera is updating

Reply:
I think it was to send a clear message immediately to the user that loading stops when camera is moved, even though that wasn't entirely true always.